src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -40,6 +42,7 @@ internal class WorkloadManifestUpdater : IWorkloadManifestUpdater
             _nugetPackageDownloader = nugetPackageDownloader;
             _sdkFeatureBand = new SdkFeatureBand(_workloadManifestProvider.GetSdkFeatureBand());
             _packageSourceLocation = packageSourceLocation;
+            _restoreActionConfig = restoreActionConfig;
```

Comment:
_restoreActionConfig is never being used in this file, is it needed? Can we assume that the nuget package downloader we're being passed is already configured correctly?